src/AutoRest.CSharp.V3/Output/Models/Serialization/Json/JsonPropertySerialization.cs

```diff
@@ -5,15 +5,17 @@ namespace AutoRest.CSharp.V3.Output.Models.Serialization.Json
 {
     internal class JsonPropertySerialization
     {
-        public JsonPropertySerialization(string name, string memberName, JsonSerialization valueSerialization)
+        public JsonPropertySerialization(string name, string memberName, JsonSerialization valueSerialization, bool? isFlattened)
```

Comment:
Nullable bool parameter since the only time the constructor is called, the `IsBag` property on Property is nullable bool. We just feed it directly into this.